docs/app/data-model/collections.md

```diff
@@ -380,3 +379,23 @@ To delete a collection, follow these steps.
 This action is permanent and irreversible. Please proceed with caution.
 
 :::
+
+## Live Preview a Collection
+
+The live preview feature allows users to instantly preview draft content without publishing or manually refreshing the
+browser.
+
+To enable this feature, navigate to the **Settings** menu and select **Data Model.** Choose the collection you want to
+allow the live preview feature.
+
+Add your URL as the **Preview URL** in the collection settings and select the `ID`. For example, the preview URL of a
+posts collection running on localhost:3000 will be `http://localhost:3000/posts/ID`.
```

Comment:
Again I think this is misleading. A user can set up any url pattern they want with dynamic values from the specific item they are previewing. I think this should be mentioned here.